python/ray/rllib/evaluation/tf_policy_graph.py

```diff
@@ -110,10 +110,28 @@ def __init__(self,
         self._batch_divisibility_req = batch_divisibility_req
 
         self._optimizer = self.optimizer()
-        self._grads_and_vars = [(g, v)
-                                for (g, v) in self.gradients(self._optimizer)
-                                if g is not None]
-        self._grads = [g for (g, v) in self._grads_and_vars]
+        grads_and_vars = self.gradients(self._optimizer)
```

Comment:
All the changes in this file are just to support multiple optimizers/losses/grads_and_vars in the `TFPolicyGraph`. I'm not quite sure what the right interface for this would be, or if supporting multiple optimizers is needed at all (I think it is). @richardliaw @ericl Do you guys have any thoughts on this?

Reply:
If they're all of the same type (e.g., Adam), having a single one + loss coefficients should work right?